doc/whats_new/v0.20.rst

```diff
@@ -175,13 +175,15 @@ Changelog
 Miscellaneous
 .............
 
+- |Fix| Deprecated using joblib from `sklearn.utils` and removed all
+  mentions to `sklearn.externals.joblib`. Only `parallel_backend` and
+  `register_parallel_backend` are accessible in `utils`.
```

Comment:
Very minor, maybe,
"""
Removed all mentions of `sklearn.externals.joblib`, and deprecated joblib methods exposed in
`sklearn.utils`, except for :func:`utils.parallel_backend` and :func:`utils.register_parallel_backend`.
"""
Because technically all the other methods are also still acessible...